web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -402,12 +402,11 @@ function joinup_core_post_update_update_update_eira_terms() {
   $filename = DRUPAL_ROOT . '/../resources/fixtures/EIRA_SKOS.rdf';
   $graph->parseFile($filename);
 
-  // Copied over from \Drupal\rdf_entity\Entity\RdfEntitySparqlStorage::insert.
```

Comment:
This update function is misspelled, let's rename it to `joinup_core_post_update_update_eira_terms()`.

Reply:
@pfrenssen there's no need. It is already updated in a PR that was not yet merged in the epic branch: https://github.com/ec-europa/joinup-dev/pull/1521.